info/container.go

```diff
@@ -197,6 +197,8 @@ type CpuStats struct {
 		// Unit: nanoseconds
 		System uint64 `json:"system"`
 	} `json:"usage"`
+	// Number of runnable threads x 1000.
+	// We multiply by thousand to avoid using floats, but preserving precision.
```

Comment:
This is a smoothed avg of number of running threads. I'll update the comment to make it clear.